src/main/kotlin/com/mjucow/eatda/presentation/store/Store.kt

```diff
@@ -0,0 +1,43 @@
+package com.mjucow.eatda.presentation.store
+
+import com.mjucow.eatda.common.dto.CursorPage
+import com.mjucow.eatda.domain.store.service.command.dto.MenuCreateCommand
+import com.mjucow.eatda.domain.store.service.command.dto.StoreCreateCommand
+import com.mjucow.eatda.domain.store.service.command.dto.StoreUpdateCommand
+import com.mjucow.eatda.domain.store.service.query.dto.MenuList
+import com.mjucow.eatda.domain.store.service.query.dto.StoreDetailDto
+import com.mjucow.eatda.domain.store.service.query.dto.StoreDto
+import com.mjucow.eatda.presentation.common.ApiResponse
+import com.mjucow.eatda.presentation.common.example.StoreDtosApiResponse
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+
+@Tag(name = "가게 API", description = "가게를 관리해주는 API")
+interface Store {
```

Comment:
다른 친구들도 `ApiPersentation` 뒤에 붙여주세요
```suggestion
interface StoreApiPersentation {
```

Reply:
넵